src/net/if_.rs

```diff
@@ -215,6 +215,15 @@ libc_bitflags!(
         /// Don't exchange routing info.
         #[cfg(any(target_os = "solaris"))]
         IFF_NORTEXCH;
+        /// Do not provide packet information
+        #[cfg(any(target_os = "linux"))]
+        IFF_NO_PI as i32;
```

Comment:
This should be `c_int` for all 3 of these to match the enum declaration.

Reply:
it's done